Lib/test/test_unittest/testmock/testmock.py

```diff
@@ -115,6 +115,26 @@ def f(): pass
         with self.assertRaises(TypeError):
             mock()
 
+    def test_create_autospec_should_be_configurable_by_kwargs(self):
+        """If kwargs are given to set side_effect & return_value, the function
+        must configure the parent mock during initialization."""
+        class Result:
+            def get_result(self):
```

Comment:
Please make sure this method is called at some point or add a `# pragma: no cover` to the end.
(code coverage is checked in the backport)

Reply:
Removed the class and reusing the class which is already defined at top of the module.